etc/kayobe/environments/ci-tenks/kolla/config/bifrost/bifrost.yml

```diff
@@ -0,0 +1,5 @@
+# yamllint disable-file
+---
+# Don't build an IPA deployment image, instead download upstream images.
```

Comment:
nit: I wouldn't say upstream here, as that implies they're not our images. 
With `stackhpc_ipa_image_bifrost_enabled` defaulting to to true, it will be our own ones